shopBuddy/shopBuddy/Item+Persistent.h

```diff
@@ -0,0 +1,30 @@
+//
+//  Item+Persistent.h
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/21/22.
+//
+
+#import "Item.h"
+#import "Parse/Parse.h"
+#import "ShoppingList.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Item (Persistent)
+
+@property (nonatomic,copy) NSString *objectID;
+
++ (void) createObject:(PFObject *)object;
+
++ (Item *) createItemWithDictionary:(NSDictionary *)dictionary;
+
++ (Item *) hydrateItemFromPFObject: (PFObject *)object;
+
+- (PFObject *) hydratePFObjectFromItemWithListObject: (PFObject *)list;
+
++ (PFObject *)populateObjectFromPointerObject: (PFObject *)object;
```

Comment:
I don't think this method belongs to the `Item`, there is no usage of self in its implementation.